src/context/UseVillagers.js

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect } from 'react';
```

Comment:
This isn't a context provider so I actually like to make custom hooks in a /hooks folder.